ixdiagnose/plugins/network.py

```diff
@@ -29,6 +29,15 @@ class Network(Plugin):
                 MiddlewareCommand('route.system_routes'),
             ]
         ),
+        MiddlewareClientMetric(
+            'rdma_config', [
+                MiddlewareCommand('rdma.capable_protocols', result_key='capable_protocols'),
+                MiddlewareCommand('rdma.get_card_choices', result_key='card_choices'),
+                MiddlewareCommand('rdma.get_link_choices', [True], result_key='all_link_choices'),
+                MiddlewareCommand('rdma.get_link_choices', result_key='link_choices'),
```

Comment:
What does link_choices show that all_link_choices doesn't?

Reply:
It makes it clear which links are currently configured (e.g. as a regular network interface).